### ISSUE TITLE
suggestion: meta data in (block specific settings)

### ISSUE THREAD

**Issue by Drakkxs** [Laike-Endaril/Controlled-Burn#7]
for example
block-specific-settings
```
        minecraft:dirt:0,1,0
        minecraft:dirt:1,7,0
        minecraft:dirt:2,6,2 
```
dirt two and one are now flammable, but dirt zero is not, this is what I am suggesting
but currently the mod will do this:
[Client thread/INFO] [STDERR]: [com.fantasticsource.controlledburn.FireData:update:42]: Block(s) not found: minecraft:dirt:0
[Client thread/INFO] [STDERR]: [com.fantasticsource.controlledburn.FireData:update:42]: Block(s) not found: minecraft:dirt:1
[Client thread/INFO] [STDERR]: [com.fantasticsource.controlledburn.FireData:update:42]: Block(s) not found: minecraft:dirt:2
and these values would be ignored
can you please fix/add this?

**Comment by Laike-Endaril** [comment]
Funny thing...I was gone from modding for 3~4 years and JUST came back to it less than a week ago.  I'm still getting used to it again, so I'm mostly working my way through 2 specific mods (1 new, 1 old) before I go through the rest of my old mods.

Anyway, quite ironically, your recent closure of this issue came up as a notification, bringing my attention to it.  If you're still interested in this feature, post here to let me know.

**Comment by Laike-Endaril** [comment]
Well, I looked into this because it sounded interesting, but a couple important override methods I could expect other mods to call, getEncouragement() and getFlammability(), only accept a Block object as an argument.  I could add new versions of them that additionally accept the meta, but other mods would still end up calling the version without meta.

In fact, even the Block class in vanilla makes a call to those...

Basically, there's no good way to do this without causing bugs, so I hope you're not too interested in it anymore!

**Comment by Drakkxs** [comment]
Thank you for the consideration, I appreciate it.
